api/server/testrun.go

```diff
@@ -96,7 +95,7 @@ func (tapi ToDDApi) TestData(w http.ResponseWriter, r *http.Request) {
 
 		// Make sure UUID string actually contains something
 		if len(testUuid[0]) > 0 {
-			testData = tdb.DatabasePackage.GetCleanTestData(testUuid[0])
+			testData, _ = tapi.tdb.GetCleanTestData(testUuid[0])
```

Comment:
As with elsewhere, it would be nice to see the error handled here. Perhaps return ```{}```?